README.md

```diff
@@ -123,7 +123,7 @@ Example command:
 
 ### Setup
 
-Clone this repository to your desired folder:
+Clone this repository to your desired folder: `git clone https://github.com/binyamolango/vet_clinic`
```

Comment:
- [ ] In the README file please implement the following changes for a professional repo. 🙏 
1. Kindly remove the broken live demo link as it's not necessary for this milestone.
2. Kindly fill out the empty sections with project-related information.
3. Kindly fix the broken MIT link.